src/Makefile.am

```diff
@@ -69,3 +85,5 @@ internal_tests_ec_internal_test_SOURCES = \
 		    include/s390_rsa.h include/s390_sha.h include/test_vec.h \
 		    include/rng.h ../test/testcase.h
 endif
+
+.PHONY: hmac-file hmac-file-lnk
```

Comment:
Missing newline at end of file

Reply:
Newline added. Didn't know there is one required.